gbn/timeout_manager.go

```diff
@@ -7,13 +7,14 @@ import (
 )
 
 const (
-	defaultTMHandshakeTimeout = 100 * time.Millisecond
-	defaultTMResendTimeout    = 100 * time.Millisecond
-	minimumResendTimeout      = 100 * time.Millisecond
-	defaultTMFinSendTimeout   = 1000 * time.Millisecond
-	defaultResendMultiplier   = 5
-	DefaultTMSendTimeout      = math.MaxInt64
-	DefaultTMRecvTimeout      = math.MaxInt64
+	defaultTMHandshakeTimeout     = 100 * time.Millisecond
```

Comment:
i think this whole commit can be squashed with the previous one

Reply:
Ok will keep that in mind for the future so I know where to draw the line of where to create a separate commit for something or not :).